ui/src/Upload.vue

```diff
@@ -52,13 +49,11 @@
     
         <div v-if="session">
             <div v-if="session.status == 'created'">
-                <p>
-                    <h3>
-                        Uploading
-                        <font-awesome-icon icon="spinner" pulse/>
-                    </h3>
-                    <small>Please do not close/refresh this page until all files are uploaded.</small>
-                </p>
+                <h3>
+                    Uploading
+                    <font-awesome-icon icon="spinner" pulse/>
+                </h3>
+                <small>Please do not close/refresh this page until all files are uploaded.</small>
```

Comment:
i removed this originally because i was getting template parsing errors:

![image](https://github.com/brainlife/ezbids/assets/26612023/f2a5d6dd-62dc-4d2f-bb88-20507aa61e26)